autoload/lsp/diag.vim

```diff
@@ -210,10 +210,16 @@ def ShowDiagInPopup(diag: dict<any>)
     # The line number is outside the last line in the file.
     dlnum = lastline
   endif
+  # Make sure column is not outside the character in line.
   if dlcol < 1
-    # The column is outside the last character in line.
-    dlcol = ltext->len() + 1
+    dlcol = 1
+  elseif dlcol > ltext->len()
+    dlcol = ltext->len()
```

Comment:
`dlcol` is character count, while `len(` returns byte count, you might be able to use `strcharlen` instead.

Reply:
This isn't resolved.